SMBLibrary.Tests/SMB2EncryptionTests.cs

```diff
@@ -7,11 +7,12 @@
 using System;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SMBLibrary.SMB2;
+using SMBLibrary.SMB2.Encryption;
 using Utilities;
 
 namespace SMBLibrary.Tests
 {
-    [TestClass]
+	[TestClass]

```

Comment:
Why did you change the indentation here?

Reply:
Thanks, looks like my tab/spacing settings are all messed up. I will revert these.